drop envsubst dep

markstos opened the pull request:
The envsubst command is often not installed by default on minimal distros.

However, the same result can be accomplished by calling a shell `eval` command
which will cause the command to be evaluated with all the currently set environment
variables considered.

This change has not been tested yet.

quantonganh commented:
I was afraid that there might be a security issue if we expand all environment variables by default, but since we manage our configuration file, I think it's okay to use `eval`.

Could you please resolve conflicts?

markstos committed:
drop envsubstr dep

The envsubst command is often not installed by default on minimal distros.

However, the same result can be accomplished by calling a shell `eval` command
which will cause the command to be evaluated with all the currently set environment
varialbles considered.

markstos commented:
Conflicts resolved.

Next, perhaps one of the `jq` or `yq` deps can be dropped, since it seems their functionality overlaps.

markstos commented:
Conflicts resolved again after `jq` dep was dropped.

quantonganh commented on helix-wezterm.sh:
```diff
@@ -155,6 +155,5 @@ if [ "$act" != "null" ]; then
 
-  expanded_command=$(echo $command | envsubst '$WEZTERM_PANE,$basedir,$binary_output,$buffer_name,$cursor_line,$interface_name,$test_name,$session,$entry')
-  echo "$expanded_command\r" | $send_to_pane
+  eval echo "$command\r" | $send_to_pane
```
Please remove `\r` as it is parsed by the shell before `eval`, causing the command to be turned into something like:

```sh
+ eval echo 'tig blame $buffer_name +$cursor_line\r'
++ echo tig blame helix-wezterm.sh +160r
```

This results in an error:

```sh
$ tig blame helix-wezterm.sh +160r
tig: Invalid number of options to blame
```

markstos replied:
Commit has been amended to resolve this issue.
